### ISSUE TITLE
"initMap is not a function" when trying the first (simplest?) maps API sample

### ISSUE THREAD

**Issue by jasonpolites** [googlemaps/js-samples#645]
#### Environment details

1. Maps JavaScript API
2. Windows 10 Home
3. Chrome Version 89.0.4389.114 (Official Build) (64-bit)
3. Node/npm environment details below...

```
node --version
v15.14.0

npm --version
7.9.0
```

#### Steps to reproduce

Following the guidance at [https://developers.google.com/maps/documentation/javascript/examples/map-simple](https://developers.google.com/maps/documentation/javascript/examples/map-simple)

```
npx @googlemaps/js-samples init map-simple .
```

This command completes successfully.  Following the `README.md` found in the root of the sample, which suggests:

```
npm i
npm run dev  # development
```

Running `npm run dev` yields the following output:

```
npm run dev

> dev
> webpack-cli serve --config ./webpack.dev.js

[webpack-cli] No need to use the 'serve' command together with '{ watch: true }' configuration, it does not make sense.
i ｢wds｣: Project is running at http://0.0.0.0:8080/
i ｢wds｣: webpack output is served from /
i ｢wds｣: Content not from webpack is served from D:\[REDACTED]\public
i ｢wds｣: 404s will fallback to /index.html
i ｢wdm｣: asset index.js 891 KiB [emitted] (name: main)
asset style.css 1010 bytes [emitted] (name: main)
asset index.html 585 bytes [emitted]
Entrypoint main 892 KiB = style.css 1010 bytes index.js 891 KiB
runtime modules 1010 bytes 5 modules
modules by path ./node_modules/ 335 KiB
  modules by path ./node_modules/webpack-dev-server/ 21.2 KiB
    modules by path ./node_modules/webpack-dev-server/client/ 20.9 KiB 10 modules
    modules by path ./node_modules/webpack-dev-server/node_modules/ 296 bytes 2 modules
  modules by path ./node_modules/html-entities/lib/*.js 61 KiB 5 modules
  modules by path ./node_modules/webpack/hot/ 1.58 KiB 3 modules
  modules by path ./node_modules/url/ 37.4 KiB 3 modules
  modules by path ./node_modules/querystring/*.js 4.51 KiB 3 modules
  4 modules
modules by path ./src/ 925 bytes (javascript) 252 bytes (css/mini-extract)
  ./src/index.ts 875 bytes [built] [code generated]
  ./src/style.css 50 bytes [built] [code generated]
  css ./node_modules/css-loader/dist/cjs.js!./src/style.css 252 bytes [code generated]
webpack 5.24.3 compiled successfully in 2579 ms
i ｢wdm｣: Compiled successfully.
```

(no obvious errors)

Launching Chrome and browsing to `http://localhost:8080/` renders a blank page.  Devtools=>Console shows:

```
{
    "message": "initMap is not a function",
    "name": "InvalidValueError",
    "stack": "Error\n    at new Se (https://maps.googleapis.com/maps/api/js?key=[REDACTED]&callback=initMap&libraries=&v=weekly:82:227)\n    at Object._.Te (https://maps.googleapis.com/maps/api/js?key=[REDACTED]&callback=initMap&libraries=&v=weekly:82:337)\n    at sk (https://maps.googleapis.com/maps/api/js?key=[REDACTED]&callback=initMap&libraries=&v=weekly:152:238)\n    at https://maps.googleapis.com/maps/api/js?key=[REDACTED]&callback=initMap&libraries=&v=weekly:152:123"
}
```

API Key was generated following instructions here:

[https://developers.google.com/maps/documentation/javascript/get-api-key](https://developers.google.com/maps/documentation/javascript/get-api-key)

API Key is restricted to only the Google Maps API:

*Screenshot:*
![image](https://user-images.githubusercontent.com/764528/114260626-034fd700-998b-11eb-8f04-472db95d6c38.png)

🤷‍♂️ 

I'm sure I can debug this to figure out what's going on, but it feels like the default/simple path in the docs should "just work"

**Comment by jpoehnelt** [comment]
Thanks for reporting. I'll take a look. It might be an issue with the async/defer due to a recent change.

**Comment by jpoehnelt** [comment]
An initial workaround is to use `npm run build` and manually visit the public/index.html output.

**Comment by jpoehnelt** [comment]
Seems related to https://github.com/webpack/webpack-dev-server/issues/2484. I'll look into upgrading this dependency.

**Comment by jpoehnelt** [comment]
see the fix in #650.
